packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1125,13 +1125,20 @@ class RenderEditable extends RenderBox {
   /// When [ignorePointer] is true, an ancestor widget must respond to tap
   /// down events by calling this method.
   void handleTapDown(TapDownDetails details) {
-    _lastTapDownPosition = details.globalPosition + -_paintOffset;
+    setLastTapDownPosition(details.globalPosition);
   }
   void _handleTapDown(TapDownDetails details) {
     assert(!ignorePointer);
     handleTapDown(details);
   }
 
+  /// Sets the last tap down position.
+  ///
+  /// The input must be the global pointer position without any offset applied.
+  void setLastTapDownPosition(Offset globalPosition) {
```

Comment:
Ya, I wouldn't add this. I think this is more "correct" but we already got a pattern with the other callbacks that are already somewhat making assumptions. Better to keep the pattern.

Reply:
Ok, the pattern being that we just assume enough time has passed between tap down and the activation of the force touch? My only worry is if the user presses down fast enough that force touch start would be called before tap down, do you not think that's possible?